schedule/yast/sle/guided_btrfs/guided_btrfs_kvm@s390x_kvm.yaml

```diff
@@ -0,0 +1,46 @@
+---
```

Comment:
This file's name can change to 'guided_btrfs@s390x_kvm.yaml'

Reply:
Done.